doc/nrf/libraries/bin/bt_ll_pcft/CHANGELOG.rst

```diff
@@ -0,0 +1,26 @@
+.. _bt_ll_pcft_changelog:
+
+Changelog
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+Notable changes to this controller are documented in this file.
+
+Controller version 3322
```

Comment:
I don't understand.
The 3330 controller is moved to a new place.  And a changelog is added for the 3322 controller.

Reply:
I have added changelog for both now.